sbi/utils/mcmc_utils.py

```diff
@@ -0,0 +1,82 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+import logging
+from typing import Callable, Optional, Union
+
+from numpy import ndarray
+from torch import Tensor
+
+from sbi.samplers.mcmc import SliceSamplerSerial, SliceSamplerVectorized
+from sbi.utils.torchutils import tensor2numpy
+
+
+def get_mcmc_samples(
+    potential_fn: Callable,
+    initial_params: Union[ndarray, Tensor],
```

Comment:
I think this will be easier to use if we can use one of our initialization strategies, so the user does not need to initialize the chains themselves.